### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by cedev935** [vercel/community#4055] [Account Issue] [answered]
### Question

I am trying this the first time and still got blocked. This is a really weird user experience. There is a similar discussion thread where there is no clear instruction to solve the issue.

Help Needed -
How to resolve this issue(get myself unblocked)

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will open a ticket with the team that can help with registration issues. 🙏
